docs/user/wallets/dashcore/advanced.rst

```diff
@@ -52,36 +52,6 @@ HD Wallets
 
 .. _dashcore-hd-default:
```

Comment:
All these labels need to remain with correct paragraphs since they establish anchors for links from other pages.